syncano/connection.py

```diff
@@ -290,6 +290,8 @@ def get_response_content(self, url, response):
 
         # Validation error
         if is_client_error(response.status_code):
+            if response.status_code == 400 and 'Revision mismatch.' in content.get('non_field_errors', [''])[0]:
```

Comment:
Yep. Think about it when implementing this 'little monster' check :) I think we should create a task for this and handle this in backend and then correct this here.